include/openssl/e_os2.h

```diff
@@ -140,6 +140,8 @@ extern "C" {
 
 /* ---------------------------- HP NonStop -------------------------------- */
 # ifdef __TANDEM
+#  include <stdint.h>
```

Comment:
This seems like it would be more appropriately placed in the "Standard integer types" section of this file that starts around line 219.